src/toolbar/ToolbarButton.tsx

```diff
@@ -72,6 +85,7 @@ export const ToolbarButtonView = forwardRef<HTMLButtonElement, ToolbarButtonView
                                 onClick={onClick}
                                 className={b(null, [className])}
                                 aria-label={titleText}
+                                data-toolbar-item={id}
```

Comment:
@d3m1d0v 
We want to provide a way to reference the toolbar button via an attribute. 
What if approach might look like this:

```
    return (
        <ToolbarButtonView
            {...props}
            id={`toolbar-item-${id}`}
            qa={qa ?? `toolbar-item-${id}`}
            active={active}
            enabled={enabled}
            onClick={handleClick}
        />
    );
```